diff --git a/node_modules/wt-tracker/src/uws-tracker.ts b/node_modules/wt-tracker/src/uws-tracker.ts
--- a/node_modules/wt-tracker/src/uws-tracker.ts
+++ b/node_modules/wt-tracker/src/uws-tracker.ts
@@ -15,7 +15,9 @@
  */
 
 import { StringDecoder } from "string_decoder";
-import { App, SSLApp } from "uWebSockets.js";
+import uWebSockets from "uWebSockets.js";
+
+const { App, SSLApp } = uWebSockets;
 import type {
   HttpRequest,
   HttpResponse,
